go/felix/rules/static.go

```diff
@@ -429,6 +450,87 @@ func (r *DefaultRuleRenderer) StaticNATOutputChains(ipVersion uint8) []*Chain {
 	}}
 }
 
+func (r *DefaultRuleRenderer) StaticRawTableChains(ipVersion uint8) []*Chain {
+	return []*Chain{
+		r.failsafeInChain(),
+		r.failsafeOutChain(),
```

Comment:
You can have explicit DROPs in your policy even if there aren't any implicit "end of xyz" DROPs?